src/datreant/core/tests/test_locks.py

```diff
@@ -13,7 +12,7 @@
 def pokefile(treantfilepath, string):
     """Add a number of tags to a Treant."""
     treant = Treant(treantfilepath)
-    treant.tags.add(*["{}_{}".format(string, i) for i in range(100)])
+    treant.tags.add(*["{}_{}".format(string, i) for i in range(10)])
```

Comment:
Yes a sort of speedup.

join on linux system waits really that all task are ready. In windows the function don't wait...

http://stackoverflow.com/questions/20914828/python-multiprocessing-pool-join-not-waiting-to-go-on

For the 1000 Task my system gets 900 task before the test ask how many results there are..so I reduced the nummer to 100 tasks. Really a WIN think about threads and multitasking, pool.join() is not OS save.